tests/unit/certGen.unitTest.ts

```diff
@@ -6555,7 +6555,7 @@ describe("cert-gen", () => {
                 });
 
                 context("and lambda-to-lambda calls were unsuccessful", () => {
-                    it("should return a PRS payload without bodyMake, bodyModel and odometer history", async () => {
+                    it("should return a PRS payload without, bodyModel and odometer history", async () => {
```

Comment:
Appreciate that it has nothing to do with your changes, but can you remove the unused imports in `CertificateGenerationService.unitTest.ts` as well if you get a chance please as it's flagging on my IDE.